code/modules/events/supermatter_surge.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/supermatter_surge
 	name = "Supermatter Surge"
 	typepath = /datum/round_event/supermatter_surge
-	weight = 20
+	weight = 15
```

Comment:
Undocumented change.

Reply:
its in the description but i did the changelog too now